### PR TITLE
Improve docs styles

### PR DESCRIPTION
**Pull request by timowest**

### CODE REVIEW

**Comment by Shredder121** [comment]
If you still remember, could you summarize the changes in the CSS?

**Comment by timowest** [comment]
* More top padding
* Larger font sizes for TOC titles

**Comment by timowest** [comment]
@johnktims @Shredder121 Is this ok to merge?

**Comment by Shredder121** [comment]
I haven't really looked at the end result closely enough to suggest enhancements, but it looks good so I vote yes.

**Comment by johnktims** [comment]
I like the ToC a lot more. I'll merge this.
